app/views/dashboard/_discovery_widget_host_list.html.erb

```diff
@@ -12,11 +12,11 @@
       <tr>
         <td><%= link_to h(host.name), discovered_host_path(host) %></td>
         <td><%= host.ip %></td>
-        <td><%= turn_zero_to_not_available(host.discovery_attribute_set.cpu_count) %></td>
-        <td><%= number_to_human_size(turn_zero_to_not_available(host.discovery_attribute_set.memory * 1024 * 1024)) %></td>
-        <td><%= turn_zero_to_not_available(host.discovery_attribute_set.disk_count) %></td>
-        <td><%= number_to_human_size(host.discovery_attribute_set.disks_size) %></td>
+        <td><%= turn_zero_to_not_available(host.discovery_attribute(:cpu_count)) %></td>
```

Comment:
currently discovery only has api tests that check if the response is redirect/success and not tests on which attributes are received in the response